core/object/script_language.h

```diff
@@ -53,7 +53,9 @@ class ScriptServer {
 	static int _language_count;
 	static bool scripting_enabled;
 	static bool reload_scripts_on_save;
-	static SafeFlag languages_finished; // Used until GH-76581 is fixed properly.
+	static bool languages_finished; // Used until GH-76581 is fixed properly.
```

Comment:
```suggestion
	static bool languages_finished;
```
Unless we still consider it to not be fixed

Reply:
I think it is not fixed until main loop order is not reworked. Maybe we should move the comment to `language_initialized` variable ?

Reply:
That bug was for thread safety wasn't it? But then the comment should be left in the `.cpp` file and not just here

Reply:
It is more than thread safety. It is because ThreadPoolWorkers start before languages are initialized, and so `thread_enter` is called when languages are not init. We can add thread safety, this won't solve the problem (hence why there is the boolean flag).